models/models/music_model.py

```diff
@@ -65,3 +65,10 @@ def load(self, path: Path) -> None:
         Loads the model from `path`.
         """
         raise NotImplementedError
+
+    @abstractmethod
+    def generate(self, seed: int | list[int] | None = None) -> Path:
```

Comment:
IMO it should take path as a param

Reply:
Then return the same path? Or nothing to return?

Reply:
Return nothing